### PR TITLE
Remove mention of temporary from discoveryProperties

### PR DESCRIPTION
**Pull request by sancar**
fixes https://github.com/hazelcast/hazelcast/issues/15996

### CODE REVIEW

**Comment by sancar** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java [hunk 1]
```diff
@@ -930,16 +930,14 @@ private int getWhenNoSSLDetected() {
...
+     * <p>Enables the Discovery SPI lookup over the old native implementations.</p>
      * <p>Discovery SPI is <b>disabled</b> by default</p>
      */
     public static final HazelcastProperty DISCOVERY_SPI_ENABLED
             = new HazelcastProperty("hazelcast.discovery.enabled", false);
 
     /**
-     * <p>Enables the Discovery Joiner to use public ips from DiscoveredNode. This property is temporary and will
-     * eventually be removed when the experimental marker is removed.</p>
+     * <p>Enables the Discovery Joiner to use public ips from DiscoveredNode.</p>
```
Comment (ihsandemir) [inline hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java]: IPs

File: hazelcast/src/main/java/com/hazelcast/client/properties/ClientProperty.java [hunk 2]
```diff
@@ -107,16 +107,14 @@
             = new HazelcastProperty("hazelcast.client.invocation.backoff.timeout.millis", -1, MILLISECONDS);
 
     /**
-     * <p>Enables the Discovery SPI lookup over the old native implementations. This property is temporary and will
-     * eventually be removed when the experimental marker is removed.</p>
```
Comment (ihsandemir) [inline hazelcast/src/main/java/com/hazelcast/client/properties/ClientProperty.java]: What is old native impl? is it clear?

**Comment by sancar** [comment]
Merging without waiting for the pr builder since this is just a javadoc change.

**Pull request merged**